src/operator-docs/creation/fromPromise.ts

```diff
@@ -1,6 +1,44 @@
 import { OperatorDoc } from '../operator.model';
 
 export const fromPromise: OperatorDoc = {
-  'name': 'fromPromise',
-  'operatorType': 'creation'
+  name: 'fromPromise',
+  operatorType: 'creation',
+  signature:
+    'public static fromPromise(promise: Promise<T>, scheduler: Scheduler): Observable<T>',
+  parameters: [
+    {
+      name: 'promise',
+      type: 'Promise<T>',
+      attribute: '',
+      description: 'The promise to be converted.'
+    },
+    {
+      name: 'scheduler',
+      type: 'Scheduler',
+      attribute: 'optional',
+      description:
+        'An optional IScheduler to use for scheduling the delivery of the resolved value (or the rejection).s'
```

Comment:
Think there's a small typo at end of sentence `(or the rejection).s`

Reply:
Ah yeah, you're right. Thanks for the hint. Will fix it this afternoon.